consensus/misc/rewards.go

```diff
@@ -36,25 +36,8 @@ func calculateQiReward(header *types.Header) *big.Int {
 	return big.NewInt(1000)
 }
 
-func CalculateRewardForQi(header *types.Header) map[uint8]uint8 {
-	rewardFromDifficulty := new(big.Int).Add(types.Denominations[types.MaxDenomination], types.Denominations[10])
-	return findMinDenominations(rewardFromDifficulty)
-}
-
-func CalculateRewardForQiWithFees(header *types.Header, fees *big.Int) map[uint8]uint8 {
-	rewardFromDifficulty := new(big.Int).Add(types.Denominations[types.MaxDenomination], types.Denominations[10])
-	reward := new(big.Int).Add(rewardFromDifficulty, fees)
-	return findMinDenominations(reward)
-}
-
-func CalculateRewardForQiWithFeesBigInt(header *types.Header, fees *big.Int) *big.Int {
```

Comment:
I'm confused. Dont we need these?

Reply:
No, they are replaced with @mechanikalk 's reward function